cli/cmd/cluster-apply.go

```diff
@@ -80,6 +80,18 @@ func runClusterApply(cmd *cobra.Command, args []string) {
 	fmt.Printf("\nYour configurations are stored in %s\n", assetDir)
 
 	kubeconfigPath := assetsKubeconfig(assetDir)
+
+	cu := controlplaneUpdater{
```

Comment:
IMO this commit message is poorly written. This commit **adds update support for bootstrap-secrets controlplane component**, which implementation detail is that it should occur before checking if cluster nodes are available.